app/uk/gov/hmrc/incometaxsessiondata/domain/models/Session.scala

```diff
@@ -16,16 +16,43 @@
 
 package uk.gov.hmrc.incometaxsessiondata.domain.models
 
-import play.api.libs.json.{Format, Json}
+import play.api.libs.functional.syntax.{toFunctionalBuilderOps, unlift}
+import play.api.libs.json.{OFormat, __}
+import uk.gov.hmrc.mongo.play.json.formats.MongoJavatimeFormats
+
+import java.time.Instant
 
 case class Session(sessionID: String,
-                   mtditid: String,
-                   nino: String,
-                   saUtr: Option[String],
-                   clientFirstName: Option[String],
-                   clientLastName: Option[String],
-                   userType: String)
+                   sessionData: Option[SessionData] = None,
```

Comment:
if we can stick to original structure for the Session case class; 
if there is nothing found in mongo by given sessionId we would simply return Option[Session]